### PR TITLE
Adds liked foods and changes some dietary needs.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd
rcsadd: people now also like food, true magic.
tweak: lizards now dislike grain and dairy
tweak: ports some of the stuff i did for my TG branch of this PR
/:cl:

aka the "floyd didnt push his final update to his PR" PR.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 	roundstart = 1
 	skinned_type = /obj/item/stack/sheet/animalhide/human
 	disliked_food = GROSS
+  liked_food = JUNKFOOD | FRIED
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -53,6 +54,7 @@
 	roundstart = 0
 	var/last_eat_message = -STATUS_MESSAGE_COOLDOWN //I am here because flies
 	disliked_food = null //atleast they got that going for them
+  liked_food = GROSS
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 3]
```diff
@@ -116,7 +118,8 @@
 	highpressure_mod = 0.75
 	lowpressure_mod = 0.75
 
-	disliked_food = FRUIT //Lizards like eating shit and trash, they don't like fruit because its ripe and pure.
+	disliked_food = DAIRY | GRAIN
+  liked_food = GROSS
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 4]
```diff
@@ -521,7 +524,8 @@ datum/species/lizard/before_equip_job(datum/job/J, mob/living/carbon/human/H)
 	var/last_light_level = 0
 	var/last_light_message = -STATUS_MESSAGE_COOLDOWN
 	var/last_plantbgone_message = -STATUS_MESSAGE_COOLDOWN
-	disliked_food = MEAT | DAIRY | GRAIN //he's vegan
+	disliked_food = MEAT | DAIRY //he's vegan
+  liked_food = VEGETABLES | FRUIT
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 5]
```diff
@@ -1000,8 +1004,9 @@ datum/species/lizard/before_equip_job(datum/job/J, mob/living/carbon/human/H)
 	var/datum/action/innate/split_body/slime_split
 	var/datum/action/innate/swap_body/body_swap
 
-	disliked_food = null
-	toxic_food = MEAT | VEGETABLES | FRUIT | GRAIN | DAIRY //they like toxins though
+	disliked_food = FRUIT
+  liked_food = TOXIC
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 6]
```diff
@@ -1163,6 +1168,7 @@ GOLEMS HAVE BEEN MOVED TO THEIR OWN MODULE
 	specflags = list(NOBREATH,RESISTTEMP,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NOHUNGER,EASYDISMEMBER,EASYLIMBATTACHMENT)
 	mutant_organs = list(/obj/item/organ/tongue/bone)
 	disliked_food = null
+  liked_food = RAW | MEAT | GROSS
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 7]
```diff
@@ -1181,6 +1187,7 @@ GOLEMS HAVE BEEN MOVED TO THEIR OWN MODULE
 	mutant_organs = list(/obj/item/organ/tongue/zombie)
 	speedmod = 2
 	disliked_food = null
+  liked_food = RAW | MEAT | GROSS
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 8]
```diff
@@ -1259,6 +1266,7 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 	heatmod = 2
 	speedmod = 1
 	disliked_food = GROSS
+  liked_food = RAW | VEGETABLES
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: tab dis

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 9]
```diff
@@ -29,7 +29,6 @@
 /obj/item/weapon/reagent_containers/food/snacks/candy_corn
 	name = "candy corn"
 	desc = "It's a handful of candy corn. Can be stored in a detective's hat."
-	junkiness = 20
```
Comment (Super3222) [inline code/modules/food_and_drinks/food/snacks_other.dm:32]: hmmm

**Review by Super3222 — CHANGES_REQUESTED** [review]
tab dis = fix your tabbing

**Comment by ghost** [comment]
where the hell did muh tabs go

File: code/modules/food_and_drinks/food/snacks_other.dm (hunk 9 above)
Comment (ghost) [reply code/modules/food_and_drinks/food/snacks_other.dm:32]: that shouldn't have junkiness, i accidentaly gave it junkiness in my other PR.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 10]
```diff
@@ -116,7 +118,8 @@
 	highpressure_mod = 0.75
 	lowpressure_mod = 0.75
 
-	disliked_food = FRUIT //Lizards like eating shit and trash, they don't like fruit because its ripe and pure.
+	disliked_food = DAIRY | GRAIN
+  	liked_food = GROSS
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: this should be the last one.

**Comment by ghost** [comment]
this is why i dont use the git editor that much. it hates indents.

**Review by Super3222 — APPROVED** [review]

**Pull request merged**